cmake/find_modules/FindMySQL.cmake

```diff
@@ -28,8 +28,17 @@ if (DEFINED VCPKG_TARGET_TRIPLET)
   endif()
 
   if (${FOUND_VAR})
-    set(MySQL_FOUND TRUE)
```

Comment:
Just to check: is removing this line intentional?

Reply:
Yes. This variable is set at the bottom of the file by `find_package_handle_standard_args`